dll/win32/comdlg32/lang/cdlg_Pt.rc

```diff
@@ -288,7 +288,7 @@ FONT 8, "MS Shell Dlg"
     CONTROL "", IDC_COLOR_GRAPH, "STATIC", WS_BORDER | SS_SIMPLE | WS_TABSTOP | WS_GROUP, 152, 4, 118, 116
     CONTROL "", IDC_COLOR_LUMBAR, "STATIC", SS_SIMPLE | WS_TABSTOP | WS_GROUP, 278, 4, 8, 116
     CONTROL "", IDC_COLOR_RESULT, "STATIC", SS_SIMPLE | WS_TABSTOP | WS_GROUP, 152, 124, 40, 26
-    DEFPUSHBUTTON "&OK", IDOK, 4, 167, 50, 14, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP //FIXME:accelerator collision &O
+    DEFPUSHBUTTON "OK", IDOK, 4, 167, 50, 14, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP
```

Comment:
correct for the pick-color-dlg. It doesn't have an accel on the ok button in XPSP3.

Reply:
Yep usually OK and Cancel won't have these. (Since this job is done by the ENTER and ESC keys already.)